HLTrigger/Configuration/python/customizeHLTforJetCleaning.py

```diff
@@ -0,0 +1,44 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
Please remove this file. TSG normally doesn't keep/maintain utilities specific to any one POG/PAG, unless there are special needs (and this does not look like one).

Reply:
Implemented in [7547ef9](https://github.com/cms-sw/cmssw/pull/37539/commits/7547ef952a88ab8f3fb13634dabd73d0efc127ea)